test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -54,6 +54,13 @@ def wrapper(self):
         return func(self)
     return wrapper
 
+def skipIfNoEmbed(func):
```

Comment:
@houseroad parameters for onnx and pytorch won't match for resnet model when fusing Conv and Batchnorm in the eval mode, and debug_embed_params test will fail because of that . One solution is to create a wrapper to skip debug_embed_params test. What do you think of this solution?

Reply:
Could you point me to where it fails?